pkgs/applications/office/ledger-autosync/default.nix

```
@@ -0,0 +1,47 @@
+{ stdenv, python3Packages, fetchFromGitHub, ledger, hledger }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "ledger-autosync";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "egh";
+    repo = "ledger-autosync";
+    rev = "d2c0e403b5a5cc8f312a0401210124eaf40a887b";
```

This repo has a tag `v1.0.0` so you can do
```suggestion
    rev = "v${version}";
```

This also makes updating a breeze.

---

Done